src/config/defaults.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  HOST: "0.0.0.0",
+  PORT: "8000",
+  BASE_URL: "http://localhost:8000",
+  WEBPACK_DEV_URL: "http://localhost:8001",
+  NODE_ENV: "development",
+  UBUNTU_SSO_URL: "https://login.ubuntu.com",
+  OPENID_VERIFY_URL: "http://localhost:8000/login/verify",
+  LP_API_URL: "https://api.launchpad.net",
+  STORE_API_URL: "https://myapps.developer.ubuntu.com/dev/api",
+  STORE_DEVPORTAL_URL: "https://myapps.developer.ubuntu.com/dev",
+  GITHUB_API_ENDPOINT: "http://localhost:4000/github",
+  GITHUB_AUTH_LOGIN_URL: "http://localhost:4000/github/login/oauth/authorize",
+  GITHUB_AUTH_VERIFY_URL: "http://localhost:4000/github/login/oauth/access_token",
+  GITHUB_AUTH_REDIRECT_URL: "http://localhost:8000/auth/verify",
+  GITHUB_AUTH_CLIENT_ID: "389a7b4c2ade662c0bf4",
+  GITHUB_AUTH_CLIENT_SECRET: "ea482b47a4830d38a838f48be43ab28002c33187",
```

Comment:
No. They are placeholder values that I created on my own account and immediately revoked. They have not been used by either the staging or the production sites.

I will be changing them to be obvious placeholder values.